core/trino-main/src/main/java/io/trino/execution/CommentTask.java

```diff
@@ -39,6 +39,7 @@
 import static io.trino.spi.StandardErrorCode.NOT_SUPPORTED;
 import static io.trino.spi.StandardErrorCode.TABLE_NOT_FOUND;
 import static io.trino.sql.analyzer.SemanticExceptions.semanticException;
+import static java.lang.String.format;
 import static java.util.Objects.requireNonNull;
 
 public class CommentTask
```

Comment:
Would it be useful to add a TestCommentTask similar to the way we have tests for other DataDefinitionTask ?